README.md

```diff
@@ -1,18 +1,21 @@
-# Amaze File Manager
```

Comment:
Why remove the project name 🤔

Reply:
![imagen](https://user-images.githubusercontent.com/10991116/114951826-7fcf3380-9e2b-11eb-85c5-783a66d9d480.png)